packages/react/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -268,6 +274,11 @@ export interface MultiSelectProps<ItemType>
    */
   readOnly?: boolean;
 
+  /**
+   * to Show/Hide All option
+   */
+  selectAll?: boolean;
```

Comment:
Should this be prefixed with `has`?

```suggestion
  hasSelectAll?: boolean;
```

Reply:
Have renamed it to `hasSelectAll`